Ext/Accessibility/build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'org.jetbrains.dokka'
+}
+
+ext {
+    pom = [
+            publishedGroupId  : 'dev.testify',
+            artifact          : 'testify-accessibility',
+            libraryName       : 'testify-accessibility',
+            libraryDescription: 'Accessibility checks for Android Testify',
+            siteUrl           : 'https://github.com/ndtp/android-testify',
+            gitUrl            : 'https://github.com/ndtp/android-testify.git',
+            licenseName       : 'The MIT License',
+            licenseUrl        : 'https://opensource.org/licenses/MIT',
+            author            : 'ndtp'
+    ]
+}
+
+version = "$project.versions.testify"
+group = pom.publishedGroupId
+archivesBaseName = pom.artifact
+
+android {
+    compileSdkVersion coreVersions.compileSdk
+
+    lintOptions {
+        abortOnError true
+        warningsAsErrors true
+        textOutput 'stdout'
+        textReport true
+        xmlReport false
+    }
+
+    defaultConfig {
+        minSdkVersion coreVersions.minSdk
+        targetSdkVersion coreVersions.targetSdk
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        multiDexEnabled = true
+        kotlinOptions {
+            allWarningsAsErrors = true
+        }
+    }
+
+    libraryVariants.all { variant ->
+        variant.outputs.all {
+            outputFileName = "${archivesBaseName}-${version}.aar"
+        }
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
+        unitTests.all {
+            testLogging {
+                events "passed", "skipped", "failed", "standardOut", "standardError"
+                outputs.upToDateWhen { false }
+                showStandardStreams = true
+            }
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+        useIR = true
+    }
+
+    dependencies {
+        implementation project(":Library")
+
+        implementation 'com.google.code.gson:gson:2.9.0'
+        implementation "androidx.core:core-ktx:${versions.androidx.core}"
```

Comment:
Is the fix of hardcoded + centralized versions intentional?

Reply:
Ah, no. That's a mistake. Thanks for catching that @gantonious 

The Sample app is the only project that should be using direct-references to versions. All the other libraries I want a centralized place to update everything, so this should be using the `version` variables.

I'll fix this before I merge it